### ISSUE TITLE
Texture Hallucination for Large-Factor Painting Super-Resolution

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#64]
# TL;DR

In super-resolution for painting, this study reproduces details that are easily lost due to the loss of the high-frequency components by using reference image and Wavelet. In this study, we have succeeded in reproducing the "painterly quality" of the generated image in comparison with the previous study.

![Fig  1  Visual comparisons for the scaling factor of 16x (the first row) and zoom-in](https://user-images.githubusercontent.com/24524018/91972829-33c8f600-ed56-11ea-9152-2bd5dcc13fc1.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/1912.00515

# Submission Dates(yyyy/mm/dd)
2020/07/30

# Authors and institutions
Yulun Zhang, Zhifei Zhang, Stephen DiVerdi, Zhaowen Wang, Jose Echevarria, Yun Fu
- Northeastern University, USA
- Adobe Research, USA

# Methods

# Results

# Comments
ECCV2020
